use whith spave between VERSION and QEMU_PKGVERSION

Our kvm version parser expects a white space ot comma after version,
see PVE::QemuServer::kvm_user_version()

Index: new/vl.c
===================================================================
--- new.orig/vl.c
+++ new/vl.c
@@ -1913,7 +1913,7 @@ static void main_loop(void)
 
 static void version(void)
 {
-    printf("QEMU emulator version " QEMU_VERSION QEMU_PKGVERSION ", Copyright (c) 2003-2008 Fabrice Bellard\n");
+    printf("QEMU emulator version " QEMU_VERSION " " QEMU_PKGVERSION ", Copyright (c) 2003-2008 Fabrice Bellard\n");
 }
 
 static void help(int exitcode)
